src/app/public/modules/repeater/repeater.component.spec.ts

```diff
@@ -522,4 +535,56 @@ describe('Repeater item component', () => {
       });
     }));
   });
+
+  describe('with inline-form', () => {
+    let fixture: ComponentFixture<RepeaterInlineFormFixtureComponent>;
+    let el: any;
```

Comment:
Should this be typed `HTMLElement`? You may need to cast line `546` but it will still give us better intellisense later in the file.